shims/common/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -593,7 +602,9 @@ object GlutenConfig {
       ("spark.sql.orc.compression.codec", "snappy"),
       ("spark.sql.decimalOperations.allowPrecisionLoss", "true"),
       ("spark.gluten.sql.columnar.backend.velox.fileHandleCacheEnabled", "false"),
-      ("spark.gluten.velox.awsSdkLogLevel", "FATAL")
+      ("spark.gluten.velox.awsSdkLogLevel", "FATAL"),
+      ("spark.gluten.velox.s3UseProxyFromEnv", "false"),
```

Comment:
Why set them in backend conf? Looks like session conf is enough.

Reply:
Just like sdkLogLevel configuration, will be used in hive connector in velox